.pre-commit-config.yaml

```
@@ -60,4 +72,4 @@ repos:
   rev: 2.1.1
   hooks:
     - id: bashate
-      entry: bashate --error . --ignore=E006,E040,E043
+      entry: bashate --error . --ignore=E006,E040,E011,E020,E012
```

we can fix it later but i would prefer to have a comment explaining what each of the ignored errors are.

---

Updated.